.github/workflows/build.yml

```diff
@@ -36,23 +36,29 @@ jobs:
     steps:
       # Note that a duplicate of this step must be added at the top of
       # each job.
```

Comment:
This should be updated because we do not necessarily want to copy the `with` arguments for other jobs.